solc/CommandLineParser.cpp

```diff
@@ -967,6 +967,9 @@ General Information)").c_str(),
 		m_options.optimizer.enabled = (m_args.count(g_strOptimize) > 0);
 		m_options.optimizer.noOptimizeYul = (m_args.count(g_strNoOptimizeYul) > 0);
 
+		if (m_args.count(g_strOptimizeRuns))
+			m_options.optimizer.expectedExecutionsPerDeployment = m_args[g_strOptimizeRuns].as<unsigned>();
```

Comment:
For `g_strOptimizeRuns` has a default value, so this would always be true. Technically, the if condition can be removed.
One could even replace `m_args[g_strOptimizeRuns]` by `m_args.at(g_strOptimizeRuns)` in that case.

Reply:
Oh wow - I did not know that!